opentelemetry-sdk/tests/metrics/test_metrics.py

```diff
@@ -270,36 +249,34 @@ def test_bind(self):
         metric_types = [metrics.Counter, metrics.Measure]
         for _type in metric_types:
             metric = _type("name", "desc", "unit", int, meter, ("key",))
-            kvp = {"key": "value"}
-            label_set = meter.get_label_set(kvp)
-            bound_instrument = metric.bind(label_set)
+            labels = {"key": "value"}
+            key_labels = tuple(sorted(labels.items()))
```

Comment:
These lines can be moved outside of the `for` loop.

Reply:
Right, moved outside.